manifests/init.pp

```diff
@@ -27,14 +27,15 @@
   $conf_dir       = '/etc/mesos',
   # e.g. zk://localhost:2181/mesos
   $zookeeper      = '',
+  $quorum         = 1,
```

Comment:
Until now `quorum` variable could have been set via options:
```puppet
  options => {
    quorum   => 4
  }
```
But which one should take precedence? We don't want to break existing installations, so from this point of view it should be overridden by options value, if defined.